card/mod/05_standard/set/abstract/attachment.rb

```diff
@@ -117,7 +117,7 @@ def original_filename
 end
 
 def unfilled?
-  !attachment.present? && !save_preliminary_upload? && super
+  !attachment.present? && !save_preliminary_upload? && !subcards.present?
```

Comment:
change 1 of 2: consider attachment cards with content but without attachment as empty